src/htcondor_es/convert_to_json.py

```diff
@@ -528,6 +532,22 @@ def make_list_from_string_field(ad, key, split_re="\s*,?\s*", default=None):
     except (TypeError, KeyError):
         return default
 
+
+def get_creation_time_from_taskname(ad):
+    """
+    returns the task creation date as a timestamp given the task name. 
+    CRAB task names includes the creation time in format %y%m%d_%H%M%S: 
+    190309_085131:adeiorio_crab_80xV2_ST_t-channel_top_4f_scaleup_inclusiveDecays_13TeV-powhegV2-madspin-pythia8 
+    """
+    if not 'CRAB_Workflow' in ad:
+        return recordTime(ad)
+    
+    _str_date = ad.get('CRAB_Workflow').split(':')[0]
+    try:
+        return int(time.mktime(datetime.datetime.strptime(_str_date, '%y%m%d_%H%M%S').timetuple()))
+    except:
```

Comment:
Be explicit which Exceptions are caught. In this case a `ValueError` for when the string doesn't match the format and a `TypeError` when `_str_date` is an unexpected type.

Since you have the same fallback value of `recordTime(ad)` you can actually use the try/except for both fallbacks:

```python
try:
   _str_date = ad['CRAB_Workflow'].split(':')[0]
   return int(time.mktime(datetime.datetime.strptime(_str_date, '%y%m%d_%H%M%S').timetuple()))
except (KeyError, TypeError, ValueError):
   return recordTime(ad)
```

Reply:
Agree, see the f403e17c6f282fd4d2630003dfac0959a117fc41

Reply:
I just see your comment in the previous commit, now removing the extra spaces in 67672a13d4a0b1f270cd5c65bdaf21886e096117